packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -473,6 +474,24 @@ abstract class Widget extends DiagnosticableTree {
     return oldWidget.runtimeType == newWidget.runtimeType
         && oldWidget.key == newWidget.key;
   }
+
+  // Return a numeric encoding of the specific `Widget` concrete subtype.
+  // This is used to determine if a hot reload modified the type hierarchy of
```

Comment:
nit: this is used in [Element.updateChild] to determine...

Reply:
Done, but i used backticks since this isn't a dartdoc